coremark/idf_component.yml

```
@@ -1,4 +1,4 @@
-version: "1.1.0"
+version: "1.1.1"
```

Or perhaps it should be
```suggestion
version: "1.1.0~1"
```
, not sure. The upstream tag is [v1.01](https://github.com/eembc/coremark/releases/tag/v1.01)...

---

I vote for `1.1.0~1`

---

I guess too late to roll back to 1.0.1 now, I'll set it to 1.1.0~1.